test/functional/wallet_basic.py

```diff
@@ -452,7 +452,6 @@ def run_test(self):
         extra_txid = self.nodes[0].sendtoaddress(sending_addr, Decimal('0.0001'))
         assert extra_txid not in self.nodes[0].getrawmempool()
         assert extra_txid in [tx["txid"] for tx in self.nodes[0].listtransactions()]
-        self.nodes[0].abandontransaction(extra_txid)
```

Comment:
Does the test still pass without this line?

Reply:
yes